packages/insomnia-app/app/ui/components/viewers/response-web-view.js

```diff
@@ -41,9 +41,19 @@ class ResponseWebView extends React.PureComponent<Props> {
     }
 
     const { body, contentType, url } = this.props;
-    webview.loadURL(`data:${contentType},${encodeURIComponent(body)}`, {
-      baseURLForDataURL: url,
-    });
+    const bodyWithBase = body.replace('<head>', `<head><base href="${url}">`);
+    webview.loadURL(`data:${contentType},${encodeURIComponent(bodyWithBase)}`);
+
+    // NOTE: We *should* be setting the base URL like this, but there is a bug
```

Comment:
```suggestion
    // NOTE: We *should* be setting the base URL by specifying the baseURLForDataURL option, but there is a bug
```

I had to read a few times to understand "this" was referring to the example at the end of the comment, not referring to the code above.